macros/dedupe_utils.sql

```diff
@@ -1,15 +1,15 @@
 {% macro deduped_blocks(table_name) -%}
     (
-        select 
+        select
```

Comment:
Not sure if you did this on purpose, or you had some tool to help you format these.. Might be good to share with the team for auto-formatters. :D